components/tools/OmeroWeb/omeroweb/feedback/views.py

```diff
@@ -131,12 +130,13 @@ def send_comment(request):
 ##############################################################################
 # handlers
 
-
 def csrf_failure(request, reason=""):
-    logger.warn('csrf_failure: Forbidden')
-    t = template_loader.get_template("403_csrf.html")
-    c = RequestContext(request, {})
-    return HttpResponseForbidden(t.render(c))
+    """
+    Always return Json response
+    since this is accepted by browser and API users
+    """
+    error = "CSRF Error. You need to include 'X-CSRFToken' in header"
```

Comment:
Users who get this error will have been trying to POST/PUT/DELETE, and to fix the error they will need to include 'X-CSRFToken' in the header or include 'csrfmiddlewaretoken' in the POST data, right? I can add that extra info, but I don't know if there's space here to say that you can get the csrf token from your session cookie?